travis-docker-compose.yml

```diff
@@ -23,13 +28,38 @@ services:
       MICROMASTERS_SECURE_SSL_REDIRECT: 'False'
       MICROMASTERS_DB_DISABLE_SSL: 'True'
       ELASTICSEARCH_URL: elastic:9200
+      BROKER_URL: redis://redis:6379/4
+      CELERY_RESULT_BACKEND: redis://redis:6379/4
     env_file: .env
     ports:
       - "8079:8079"
     links:
       - db
       - elastic
 
+  celery:
```

Comment:
The reason why we need Celery is not for running the async task, but for these commands:
https://github.com/mitodl/micromasters/pull/2126/files#diff-bf4aef0ab77eb78aa7345a16af8ad33cR99
and 
https://github.com/mitodl/micromasters/pull/2126/files#diff-bf4aef0ab77eb78aa7345a16af8ad33cR72